### PR TITLE
JNA improve

### PR DESCRIPTION
**Pull request by StefArma**
- Added ACE items array in init.var
- Improved ACE detection (with or without medical) and categorization for misc items
- Fixed (thanks Sparker) Arsenal from Vehicle (it was duplicating or substracting too many items)
- Better categorization: INITTYPES is better done in JeroenArsenal/JNA/fn_arsenal_itemType.sqf and will be called from JeroenArsenal/JNA/fn_arsenal_loadInventory.sqf 

- RHS was causing an error because civboat was undefined variable, i just added the boat.

### CODE REVIEW

**Comment by Barbolani77** [comment]
Hi, thank you!

I need to understand what have you done with the line " 
Fix Arsenal from Vehicle + loadinv optimize"

File: JeroenArsenal/JNA/fn_vehicleArsenal.sqf [hunk 1]
```diff
@@ -676,7 +677,8 @@ switch _mode do {
 					jnva_loadout set [_index,[jnva_loadout select _index,[_item,_count]] call jn_fnc_arsenal_removeFromArray];
 					_mass = ["getMassItem",[_item,_count,_index]] call jn_fnc_vehicleArsenal;
 					jnva_loadout_mass = jnva_loadout_mass - _mass;
-					[_index, _item, _count] remoteExecCall ["jn_fnc_arsenal_addItem"];
+					//[_index, _item, _count] remoteExecCall ["jn_fnc_arsenal_addItem"];
+					[_index, _item, _count] call jn_fnc_arsenal_addItem; //Sparker: why execute it on all clients?
```
Comment (StefArma) [inline JeroenArsenal/JNA/fn_vehicleArsenal.sqf]: This line and also the line 660 above are done by sparker, it avoid execute on all clients and causing the strange number values

File: JeroenArsenal/JNA/fn_arsenal_loadInventory.sqf [hunk 2]
```diff
@@ -350,29 +308,17 @@ _containers = [_uniform,_vest,_backpack];
 
 		call {
 			if ([_itemCounts select _index, _item] call jn_fnc_arsenal_itemCount == -1) exitWith {
-				call ([
-					{player forceAddUniform _uniform;},//todo remove
```
Comment (StefArma) [inline JeroenArsenal/JNA/fn_arsenal_loadInventory.sqf]: Theese lines were repeated multiple times, they are regrouped into line 296 with the private that can be just called without repeating.

**Comment by StefArma** [comment]
I couldn't fix this .rpt error

      9:32:29 Error in expression <"_index","_item",["_amount",1]]; _array set [_index,[[_item,_amount]]]; }else{ _>
      9:32:29   Error position: <set [_index,[[_item,_amount]]]; }else{ _>
      9:32:29   Error Zero divisor
      9:32:29 File JeroenArsenal\JNA\fn_arsenal_addItem.sqf [JN_fnc_arsenal_addItem], line 1262

nor really understand what it is. because there are no items missing.

**Comment by StefArma** [comment]
Already  integrated here: https://github.com/A3Antistasi/Antistasi-WotP/commit/73131e3ca2835788b38203719d25b4b98174279c

**Pull request closed without merging**